index.html

```diff
@@ -8,6 +8,6 @@
     <title>Examples</title>
 </head>
 <body>
-    <h1>Code Reviewer Assistant!</h1>
+    <header>Code Reviewer Assistant!</header>
```

Comment:
- [ ] In order to follow the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) concepts, please make sure that you wrap the text inside a `<h1>` tag. 😄🙏